index.bs

```diff
@@ -177,23 +174,49 @@ navigator.xr.requestSession("immersive-ar").then((session) => {
 
 Environment blend mode {#environment-blend-mode}
 ---------
-The <a href="https://www.w3.org/TR/webxr/">WebXR Device API API</a> defines each {{XRSession}} has having an [=environment blending mode=] describing the behavior of imagery rendered by the session in relation to the user's surrounding environment.
+When rendering XR content, it is often useful to how the rendered pixels will be blended against the user's environment. For example, some AR devices have transparent optical displays, also known as "additive light" or "see-through" displays. On this hardware, the user may be able to see through to the real world environment in all session types. Other AR devices which use a camera stream to [=performs automatic composition|compose with the real world=] are known as "pass-through" displays and may behave differently in each {{XRSessionMode}}. Most Virtual Reality devices are not capable of [=performs automatic composition|blending rendered pixels with the real world=] capable of exhibiting {{XREnvironmentBlendMode/"opaque"}} blending behavior.
```

Comment:
> Most Virtual Reality devices are not capable of blending rendered pixels with the real world *capable of* exhibiting {{XREnvironmentBlendMode/"opaque"}} blending behavior.

I think the second "capable of" in this sentence is unintended? Perhaps replace it with a comma,

Reply:
already applied @Manishearth's suggested fix 👍